clients/src/main/java/org/apache/kafka/clients/admin/AlterConfigsResult.java

```diff
@@ -32,10 +37,16 @@
         this.futures = futures;
     }
 
+    /**
+     * Return a map from resources to futures which can be used to check the status of the operation on each resource.
+     */
     public Map<ConfigResource, KafkaFuture<Void>> results() {
         return futures;
     }
 
+    /**
+     * Return a future which succeeds only if all the alter configs succeed.
```

Comment:
Perhaps add "operation"?  So "only if all the alter configs operations succeed." ?